internal/vds/vds.h

```diff
@@ -82,6 +89,9 @@ struct response horizon(
     float yinc,
     float rot,
     float fillvalue,
+    float above,
+    float below,
+    float samplerate,
```

Comment:
No, you are right, we should put some limits on it. `0 <= above <= 500?`, ` 0 <= below <= 500?`. 
 `samplerate <= 0.1`. According to Renaud 0.1 should be plenty for both time and depth.  Need to confirm upper bound on above/below

Rotation is used in sin and cos functions so any value positiv or negative should result in a sane transformation matrix. I don't see any problem with `xinc` and `yinc` as they just translate from origin, but are bound by the size of the map. 

i.e. all of these contribute to _where_ the map/grid points are located in physical space, but not how many there are. And if the grid/map lands outside of the volume bounds because of this the caller will get a lot of fill value back. Not a problem for us